### ISSUE TITLE
[Config Support]: Go2RTC is working only partially

### ISSUE THREAD

**Issue by Remko76** [blakeblackshear/frigate#9590]
### Describe the problem you are having

I have updated to the 0.13.1 version and was having a lot of performance issues. I did notice the same thing on another system I am managing, I am suspecting it has something to do with the detect resolution. The Wansview W5 camera has a substream with a resolution of 768x432, which I also manually configured this way. Used to run fine in 0.12, but I cannot get it to work properly in 0.13.1, very high CPU usage and cannot get it lower. It does seem to be the motion detection on that camera type, since turning motion detection for another camera on (with a sub stream resolution of 640x360) does not cause high CPU load.

Anyhow....I decided to first revert to 0.12.1, which has been running very smooth for half a year now, and this seemed to help with this issue. I wanted to look at the details of the upgrade later. But.....and now comes the issue, it all worked fine when I went to bed last night, but now I cannot seem to get the stream working through go2rtc anymore.

I removed almost all cameras from my configuration so only have one left and when I go to the go2rtc dashboard and attempt to open the RTC stream for that camera, I only get it to work with a huge delay and ONLY on the substream. I do not get anything anymore for the main stream. When all my cameras are active I am only able to get something on 2 substreams, none of the mainstreams give any image or sound.

 I have tried everything I can think of:
- reboot all my routers and access points
- rebooted all my switches 
- rebooted the cameras
- reduced the config to one camera

But..I am not able to find a solution for this, the logs show nothing, it just does not work where it still did last night.

I hope anyone has an idea of what else I could try....I am pulling my hairs out over this and am close to having none left...;)

BTW...running everything in my local network, All the streams open perfectly fast in VLC.

### Version

0.12.1

### Frigate config file

```yaml
mqtt:
  host: 192.168.76.6
  user: hass
  password: Yamaha!987

database:
  path: /media/database/frigate.db

ffmpeg:
  hwaccel_args: -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format yuv420p
  output_args:
    detect: -f rawvideo -pix_fmt yuv420p
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
  # input_args:
  #   - -avoid_negative_ts
  #   - make_zero
  #   - -fflags
  #   - nobuffer+genpts+discardcorrupt
  #   - -flags
  #   - low_delay
  #   - -strict
  #   - experimental
  #   - -analyzeduration
  #   - 10000M
  #   - -probesize
  #   - 10000M
  #   - -rw_timeout
  #   - "5000000"

detectors:
  coral:
    type: edgetpu
    device: usb

birdseye:
  enabled: True
  mode: continuous


# Resolutions 
# Wansview W5: High: 1920x1080, Low: 768x432 (15fps)
# Reolink E1Pro: High: 2560 x 1440, Low: 640x360 (15fps)
# Reolink RLC410W: High: 2560 x 1440, Low: 640x360 (15fps)


go2rtc:
  streams:
    eingang:
      - rtsp://USER:PASS@192.168.76.64:554/h264Preview_01_main
      - ffmpeg:eingang#audio=opus
    eingang_sub:
      - rtsp://USER:PASS@192.168.76.64:554/h264Preview_01_sub
      - ffmpeg:eingang_sub#audio=opus

  webrtc:
    candidates:
    - 192.168.76.6:8555
    - stun:8555
##########################################################
cameras:


##########################################################
  eingang:
    ui:
      order: 1
    ffmpeg:
      inputs:
        #High Res
        - path: rtsp://127.0.0.1:8554/eingang
          roles:
            - record
        #Low Res    
        - path: rtsp://127.0.0.1:8554/eingang_sub
          roles:
            - detect
    detect:
      enabled: true
      width: 640
      height: 360
      fps: 15
    snapshots:
      enabled: true
      required_zones:
        - entry_front_door
    record:
      enabled: true
      retain:
        days: 7
        mode: motion
      events:
        retain:
          default: 10    
        required_zones:
          - entry_front_door
    motion:
      mask:
        - 0,0,39,0,33,22,0,36
        - 354,162,388,180,435,200,412,325,266,331,262,163
    zones:
      entry_carport:
        coordinates: 223,177,229,63,298,58,336,172
      entry_front_door:
        coordinates: 96,360,438,360,315,60,191,66
    objects:
      track:
        - person
        - cat
        - bird
        - car
        - umbrella
        - motorcycle
```

### Relevant log output

```shell
2024-02-02 09:02:08.162822573  [INFO] Starting Frigate...
2024-02-02 09:02:09.464919508  [2024-02-02 09:02:09] frigate.app                    INFO    : Starting Frigate (0.12.1-367d724)
2024-02-02 09:02:09.537672757  [2024-02-02 09:02:09] peewee_migrate                 INFO    : Starting migrations
2024-02-02 09:02:09.542298281  [2024-02-02 09:02:09] peewee_migrate                 INFO    : There is nothing to migrate
2024-02-02 09:02:09.569291937  [2024-02-02 09:02:09] detector.coral                 INFO    : Starting detection process: 290
2024-02-02 09:02:12.214394895  [2024-02-02 09:02:09] frigate.app                    INFO    : Output process started: 292
2024-02-02 09:02:12.214403773  [2024-02-02 09:02:09] frigate.detectors.plugins.edgetpu_tfl INFO    : Attempting to load TPU as usb
2024-02-02 09:02:12.215755221  [2024-02-02 09:02:09] frigate.app                    INFO    : Camera processor started for eingang: 307
2024-02-02 09:02:12.216962784  [2024-02-02 09:02:09] frigate.app                    INFO    : Capture process started for eingang: 326
2024-02-02 09:02:12.221123542  [2024-02-02 09:02:12] frigate.detectors.plugins.edgetpu_tfl INFO    : TPU found



2024-02-02 09:02:08.156495661  [INFO] Preparing go2rtc config...
2024-02-02 09:02:08.473947496  [INFO] Not injecting WebRTC candidates into go2rtc config as it has been set manually
2024-02-02 09:02:08.508699505  [INFO] Starting go2rtc...
2024-02-02 09:02:08.652244423  09:02:08.652 INF go2rtc version 1.2.0 linux/amd64
2024-02-02 09:02:08.652894964  09:02:08.652 INF [api] listen addr=:1984
2024-02-02 09:02:08.653811546  09:02:08.653 INF [rtsp] listen addr=:8554
2024-02-02 09:02:08.654836609  09:02:08.654 INF [srtp] listen addr=:8443
2024-02-02 09:02:08.655476281  09:02:08.655 INF [webrtc] listen addr=:8555
2024-02-02 09:02:18.161778056  [INFO] Starting go2rtc healthcheck service...
2024-02-02 09:12:58.300180580  09:12:58.300 WRN github.com/AlexxIT/go2rtc/cmd/streams/producer.go:133 > error="read tcp 172.18.0.6:47610->192.168.76.64:554: i/o timeout" url=rtsp://USER:PASS@192.168.76.64:554/h264Preview_01_sub
2024-02-02 09:12:58.322380885  09:12:58.322 WRN github.com/AlexxIT/go2rtc/cmd/streams/producer.go:133 > error="read tcp 172.18.0.6:47618->192.168.76.64:554: i/o timeout" url=rtsp://USER:PASS@192.168.76.64:554/h264Preview_01_main
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

**Comment by Remko76** [comment]
I am going to close this issue now as I seem to have sorted it. To start with....I have no hairs left....;)

I noticed when I was at another location in my house, ie. connected through one of my mesh repeaters and all of a sudden the streams worked again. I then started digging into my router and found that is somehow recognized my router somehow switched one of it's LAN ports to 100 MBIt instead of the normal Gigabit. Since this port gets splitted later through a switch and feeds most of the important devices on my network, everything was just incredibly slow. 

How did I fix it....pulled out the network cable from my router and plugged it back in and.....all of a sudden everything is back to normal. I do not understand how or why this happend, as I said, I rebooted routers and switches several times. Anyhow.....it seems to be back to normal, next I will test the upgrade to 0.13.1 again....
